packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -148,7 +149,7 @@ const defaultProps: Partial<CollectionProps<unknown>> = {
     readOnly: false,
     gap: 'xs',
     required: false,
-    getItemId: ({id}: any) => id,
+    getItemId: ({id}: any, index: number) => id,
```

Comment:
```suggestion
    getItemId: ({id}: any) => id,
```

Reply:
It isn't used in this particular function so we can remove it, as ESLint suggests
<img width="629" alt="image" src="https://github.com/user-attachments/assets/bab3216a-8886-423c-bf17-2e9c92fccc09" />